### PR TITLE
fix to use specific version of icopenshell

### PR DESCRIPTION
**Pull request by civilx64**
ref VAL-353

### CODE REVIEW

File: backend/Makefile [hunk 1]
```diff
@@ -12,7 +12,10 @@ venv:
 install: venv
 	. $(VIRTUAL_ENV)/bin/activate && \
 	pip install --upgrade pip && \
-	pip install -r requirements.txt
+	pip install -r requirements.txt && \
+    wget -O /tmp/ifcopenshell_python.zip "https://s3.amazonaws.com/ifcopenshell-builds/ifcopenshell-python-311-v0.7.0-742a9ec-linux64.zip" && \
```
Comment (aothms) [inline backend/Makefile:16]: Is it safe to assume 3.11? I normally do

    `python3 -c 'import sys;print("".join(map(str, sys.version_info[0:2])))'`

File: backend/apps/ifc_validation/checks/ifc_gherkin_rules
Comment (aothms) [inline backend/apps/ifc_validation/checks/ifc_gherkin_rules:1]: It seems this PR downgrades the submodule?

Comment (civilx64) [reply backend/apps/ifc_validation/checks/ifc_gherkin_rules:1]: The docker images start with a 3.11 image, therefore the intent is to "pin" to that version.

File: backend/Makefile (hunk 1 above)
Comment (civilx64) [reply backend/Makefile:16]: The docker images start with a 3.11 image, therefore the intent is to "pin" to that version.

**Pull request merged**